plugins/GSdx/GSRendererHW.cpp

```diff
@@ -53,7 +53,7 @@ void GSRendererHW::SetScaling() {
 	
 	//Only increase the buffer size, don't make it smaller, it breaks games (GH3)
 
-	if (!m_nativeres && m_width < (m_buffer_size * m_upscale_multiplier)){
+	if (m_upscale_multiplier !=1 && m_width < (m_buffer_size * m_upscale_multiplier)){
```

Comment:
Included https://github.com/turtleli/pcsx2/commit/65156e7b6b4cecd6428918cc47041f1028dfe6f4 , that should fix it.  Thanks for the commit